en/plugins.rst

```diff
@@ -280,6 +280,54 @@ autoloader once you've created your plugin:
 
     $ php composer.phar dumpautoload
 
+.. _plugin-routes:
+
+Plugin Routes
+=============
+
+Plugins can provide routes files containing their routes. Each plugin can
+contain a **config/routes.php** file. This routes file can be loaded when the
+plugin is added, or in the application's routes file. To create the
+ContactManager plugin routes. Put the following into
+**plugins/ContactManager/config/routes.php**::
+
+    <?php
+    use Cake\Routing\Route\DashedRoute;
+    use Cake\Routing\Router;
+
+    Router::plugin(
+        'ContactManager',
+        ['path' => '/contact-manager'],
+        function ($routes) {
+            $routes->fallbacks(DashedRoute::class);
+        }
+    );
+
+The above will connect default routes for you plugin. You can customize this
+file with more specific routes later on.
+
+Before you can access your controllers, you'll need to ensure the plugin is
+loaded and the plugin routes are loaded.  In your **config/bootstrap.php** add
+the following::
+
+    Plugin::load('ContactManager', ['routes' => true]);
+
+You can also load plugin routes in your application's routes list. Doing this
+provides you more control on how plugin routes are loaded and allows you to wrap
+plugin routes in additional scopes or prefixes::
+
+    Router::scope('/', function ($routes) {
+        // Connect other routes.
+        $routes->scope('/a', function ($routes) {
```

Comment:
May be a locale, like ``de_DE`` or may be ``api`` as it was used on https://book.cakephp.org/3.0/en/development/routing.html#using-named-routes.